src/webgl/p5.RendererGL.Retained.js

```diff
@@ -288,6 +288,21 @@ p5.RendererGL.prototype.drawBuffers = function(gId) {
   return this;
 };
 
+p5.RendererGL.prototype.drawBuffersScaled = function(
```

Comment:
I suggest a detailed private comment for library developers here, explaining how the geometries for built-in primitives are scaled in this function (and citing a few examples that operate differently... like box + torus, what assumptions are made about their dimensions before scaling) and how it might be expanded for future functionality and the missing primitive shapes we have.